### ISSUE TITLE
logging in on a headless machine throws strange error

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode-vsce#690]
run this inside a container or something headless:
```
npx vsce login
```

Get the following:
```
ERROR  Cannot autolaunch D-Bus without X11 $DISPLAY
```

I know that you can specify a PAT instead, this error could be caught and suggest that.

**Comment by joaomoreno** [comment]
Repro:

```
$ docker run -it node:lts bash
$ apt update && apt install libsecret-1-0
$ npx vsce login foo
```

**Comment by joaomoreno** [comment]
Going with the following warning: 

![Uploading image.png…]()

**Comment by joaomoreno** [comment]
Fixed by https://github.com/microsoft/vscode-vsce/commit/a33261899f54d5dc67dcda7eb4bb4b0906bee588

Released in [2.6.5](https://github.com/microsoft/vscode-vsce/releases/tag/v2.6.5)
